### PR TITLE
Clean up API response/dto

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: sdk/src/types/api/core/response.rs [hunk 1]
```diff
@@ -293,35 +315,83 @@ pub struct UtxoChangesResponse {
     pub consumed_outputs: Vec<String>,
 }
 
-/// Response of GET /api/core/v2/peers.
```
Comment (Thoralf-M) [inline sdk/src/types/api/core/response.rs:296]: Don't you want to keep this?

File: sdk/src/types/api/core/response.rs [hunk 2]
```diff
@@ -169,6 +166,19 @@ pub enum BlockResponse {
     Raw(Vec<u8>),
 }
 
+/// Describes the ledger inclusion state of a transaction.
+#[derive(Clone, Debug, Eq, PartialEq)]
```
Comment (Thoralf-M) [inline sdk/src/types/api/core/response.rs]: Can also be Copy, right?

Comment (thibault-martinez) [reply sdk/src/types/api/core/response.rs]: Sure

File: sdk/src/types/api/core/response.rs [hunk 3]
```diff
@@ -249,7 +271,7 @@ pub enum OutputResponse {
     serde(rename_all = "camelCase")
```
Comment (kwek20) [inline sdk/src/types/api/core/response.rs]: dont ned the rename_all here actualy

File: sdk/src/types/api/core/response.rs (hunk 2 above)
Comment (thibault-martinez) [reply sdk/src/types/api/core/response.rs]: Done

File: sdk/src/types/api/core/response.rs (hunk 3 above)
Comment (thibault-martinez) [reply sdk/src/types/api/core/response.rs]: Done

**Review by Thoralf-M — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Pull request merged**